backend/tournesol/tests/test_exports.py

```diff
@@ -216,69 +238,84 @@ def test_not_authenticated_can_download_public_comparisons_multiple_users(self):
         call_command("create_dataset")
-
         response = self.client.get("/exports/all/")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.headers["Content-Type"], "application/zip")
 
-        root = self.extract_export_root(response)
-
+    @export_test_override_settings
+    def test_public_dataset_contains_all_expected_files(self):
+        call_command("create_dataset")
+        response = self.client.get("/exports/all/")
         zip_content = io.BytesIO(response.content)
         with zipfile.ZipFile(zip_content, "r") as zip_file:
-            expected_files = [
-                root + "/README.txt",
-                root + "/users.csv",
-                root + "/comparisons.csv",
-                root + "/individual_criteria_scores.csv",
+            expected_filenames = [
+                "README.txt",
+                "users.csv",
+                "comparisons.csv",
+                "individual_criteria_scores.csv",
+                "collective_criteria_scores.csv",
             ]
+            filenames = [filepath.rsplit("/", 1)[-1] for filepath in zip_file.namelist()]
+            self.assertEqual(filenames, expected_filenames)
 
-            self.assertEqual(zip_file.namelist(), expected_files)
-
+    @export_test_override_settings
+    def test_export_readme_equals_resources_readme(self):
+        call_command("create_dataset")
+        response = self.client.get("/exports/all/")
+        root = self.extract_export_root(response)
+        zip_content = io.BytesIO(response.content)
+        with zipfile.ZipFile(zip_content, "r") as zip_file:
             with zip_file.open(root + "/README.txt", "r") as file:
                 content = file.read()
                 with open("tournesol/resources/export_readme.txt", "rb") as readme_file:
                     expected_content = readme_file.read()
                 self.assertEqual(content, expected_content)
 
+    @export_test_override_settings
+    def test_export_all_comparisons_equal_export_comparisons(self):
+        call_command("create_dataset")
+        response = self.client.get("/exports/all/")
+        root = self.extract_export_root(response)
+        zip_content = io.BytesIO(response.content)
+        with zipfile.ZipFile(zip_content, "r") as zip_file:
             with zip_file.open(root + "/comparisons.csv", "r") as file:
-                content = file.read()
                 expected_content = self.client.get("/exports/comparisons/").content
-                self.assertEqual(content, expected_content)
+                self.assertEqual(file.read(), expected_content)
 
-            with zip_file.open(root + "/users.csv", "r") as file:
-                content = file.read().decode("utf-8")
-                csv_file = csv.DictReader(io.StringIO(content))
-                rows = list(csv_file)
+    @export_test_override_settings
+    def test_users_with_public_comparisons_uniquely_added_to_users_export(self):
```

Comment:
The four new tests I added:
- test_public_dataset_contains_all_expected_files
- test_export_readme_equals_resources_readme
- test_export_all_comparisons_equal_export_comparisons
- test_users_with_public_comparisons_uniquely_added_to_users_export

were all part of the same test `test_not_authenticated_can_download_all_exports`